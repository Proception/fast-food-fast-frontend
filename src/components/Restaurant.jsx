import React, { Component } from 'react';
import RestaurantControls from './RestaurantControls.jsx';
import RestaurantMenu from './RestaurantMenu.jsx';

export default class Restaurant extends Component {
  render() {
    return (
      <div className="main-container">
        <h2> Restaurant Menu</h2>
        <RestaurantControls/>
        <RestaurantMenu/>
    </div>
    )
  }
}
