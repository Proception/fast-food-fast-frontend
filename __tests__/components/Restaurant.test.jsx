import React from 'react';
import { shallow } from 'enzyme';
import Restaurant from '../../src/components/Restaurant.jsx';

let wrapper;
beforeEach(() => {
  wrapper = shallow(<Restaurant/>);
});
describe('Restaurant Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
