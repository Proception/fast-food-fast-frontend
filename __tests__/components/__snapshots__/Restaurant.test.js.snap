// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Restaurant Component should match snapshot 1`] = `
<div
  className="main-container"
>
  <h2>
     Restaurant Menu
  </h2>
  <RestaurantControls />
  <RestaurantMenu />
</div>
`;
